internal/pkg/composable/providers/kubernetes/pod.go

```diff
@@ -212,7 +212,12 @@ func (p *pod) emitRunning(pod *kubernetes.Pod) {
 		if !p.managed {
 			if ann, ok := data.mapping["annotations"]; ok {
 				annotations, _ := ann.(mapstr.M)
-				hints := utils.GenerateHints(annotations, "", p.config.Prefix)
+				//We check whether the provided annotation follows the supported format and vocabulary. The check happens for annotations that start with co.elastic.hints
+				hints, err := utils.GenerateHints(annotations, "", p.config.Prefix, allSupportedHints)
+				if err != nil {
+					p.logger.Warnf("%v for pod %v in namespace", err, pod.Name, pod.ObjectMeta.Namespace)
```

Comment:
At the moment I set as warning.
This kind of errors I think must be visible with default installation

But almost every 2sec we have a warning for one pod per wrong annotation. This can add noise
What do you think?

Reply:
They will have it every two seconds until they fix it. If the only reason to get this warning is a wrong hint annotation, this will force them to fix it. 
I would leave it as a warning.